flake.nix

```diff
@@ -3,14 +3,65 @@
 
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
-    nixpkgs.url = "nixpkgs/21.11";
+    nixpkgs.url = "nixpkgs/release-22.11";
+    poetry2nix = {
+      url = "github:nix-community/poetry2nix";
```

Comment:
We pull in poetry2nix here (`master`) as the version in `nixpkgs` doesn't contain some overrides that we need.